std_msgs/Header header
float64 rho
float64 theta
string  color
int32   tag_id
